pkg/blackduck/reporting.go

```diff
@@ -23,8 +25,15 @@ func CreateSarifResultFile(vulns *Vulnerabilities, projectName, projectVersion,
 	collectedRules := []string{}
 	cweIdsForTaxonomies := []string{}
 	results := []format.Results{}
+
 	if vulns != nil && vulns.Items != nil {
 		for _, v := range vulns.Items {
+
+			isAudited := true
+			if v.RemediationStatus == "NEW" {
```

Comment:
Blackduck's remediation status of 'Remediation required' and 'Needs Review' are also considered as Unaudited. I think those checks also need to be included here.